x-pack/plugins/ml/public/application/explorer/explorer_charts/explorer_charts_container.js

```diff
@@ -79,6 +89,47 @@ function ExplorerChartContainer({
   chartsService,
 }) {
   const [explorerSeriesLink, setExplorerSeriesLink] = useState('');
+  const [mapsLink, setMapsLink] = useState('');
+
+  const {
+    services: {
+      data,
+      share,
+      application: { navigateToApp },
+    },
+  } = useMlKibana();
+
+  const getMapsLink = useCallback(async () => {
+    const initialLayers = [];
+    for (const layer in ML_ANOMALY_LAYERS) {
+      if (ML_ANOMALY_LAYERS.hasOwnProperty(layer)) {
+        initialLayers.push({
+          id: htmlIdGenerator()(),
+          type: LAYER_TYPE.GEOJSON_VECTOR,
+          sourceDescriptor: AnomalySource.createDescriptor({
+            jobId: series.jobId,
+            typicalActual: ML_ANOMALY_LAYERS[layer],
+          }),
+          style: {
+            type: 'VECTOR',
+            properties: {
```

Comment:
What do you think about using the grey color for the fill / line color for the typical layer, giving the opened map a similar appearance to the one in the Anomaly Explorer?

Reply:
Good call - updated to use grey color for typical layer in 3bfae94c05b9d5ccd1dae877e83cf7a374e59b3e